packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx

```diff
@@ -4,48 +4,69 @@ import { Button } from './buttons/Button';
 import { ScrollOnboardingVersion } from '../lib/featureValues';
 import { cloudinary } from '../lib/image';
 import AnalyticsContext from '../contexts/AnalyticsContext';
-import { AnalyticsEvent, TargetType } from '../lib/analytics';
+import { AnalyticsEvent } from '../lib/analytics';
 
-const GaussianBlur = (): ReactElement => {
-  return (
-    <div className="absolute w-52 h-52 rounded-full opacity-32 blur-2xl bg-theme-color-cabbage" />
-  );
-};
+type ScrollOnboardingVersionMap = Partial<
+  Record<ScrollOnboardingVersion, string>
+>;
 
-const versionToContainerClassName: Record<ScrollOnboardingVersion, string> = {
-  [ScrollOnboardingVersion.V1]: 'h-60',
-  [ScrollOnboardingVersion.V2]: 'h-[36rem]',
+const versionToContainerClassName = {
+  [ScrollOnboardingVersion.V1]: 'h-48 my-5',
+  [ScrollOnboardingVersion.V2]: 'h-80 m-10',
 };
-const versionToButtonClassName: Record<ScrollOnboardingVersion, string> = {
+const versionToButtonClassName: ScrollOnboardingVersionMap = {
   [ScrollOnboardingVersion.V1]: 'w-[16.25rem]',
   [ScrollOnboardingVersion.V2]: 'w-40',
 };
-const versionToButtonText: Record<ScrollOnboardingVersion, string> = {
+const versionToButtonText: ScrollOnboardingVersionMap = {
   [ScrollOnboardingVersion.V1]: 'Customize your feed',
   [ScrollOnboardingVersion.V2]: 'Start',
 };
 
+const versionToGaussianBlurClassName: ScrollOnboardingVersionMap = {
+  [ScrollOnboardingVersion.V1]: 'top-0 right-0 bottom-0 left-0 m-auto',
+  [ScrollOnboardingVersion.V2]: '',
+};
+
+const GaussianBlur = ({
+  version,
+}: {
+  version: ScrollOnboardingVersion;
+}): ReactElement => {
+  return (
+    <div
+      className={classNames(
+        'absolute w-48 h-48 rounded-full blur-2xl opacity-[0.2] bg-theme-color-cabbage',
+        versionToGaussianBlurClassName[version],
+      )}
+    />
+  );
+};
```

Comment:
This looks to be something we can reuse in other places. Let's take this out and dedicate a file. Let's just remove some of the custom classes and make it accept a `className` property for modifications.

Reply:
This was already here from the previous PR I just adjusted it. With exposed className only I think it is not reusable in this state and for sure not giving any extra value if we just add `className`.

Some of my ideas for `GaussianBlur` component:
- should in this case be called something like `GaussianBlurRadius` since this one is specific to that version of blur
- expose radius as `prop` (so people do not need to add classes and it feels more like a component and not just `classed`/`classNames` wrapper
- expose width/height
- expose oppacity
- expose color

I think this effort is better suited for another PR.

Reply:
WDYT?

Reply:
My bad probably, I do think these would work great as individual components, maybe for now simply allow classnames option?
Eventually we can abstract with the options you mention.

Reply:
Since I have time I can add it today and cut our backlog some slack 😆

Reply:
Had some issues with typescript while implementing this, I am gonna show you the details maybe in the web team meet so we can discuss, I think its a curious case which I think we don't have in the codebase yet. Probably stumbled upon it since I am used to working with JSS libraries instead of tailwind.